audio/FAudio/Portfile

```diff
@@ -18,17 +18,18 @@ long_description        an XAudio reimplementation that focuses solely on develo
 
 depends_lib-append      port:libsdl2
 
-checksums               rmd160  947c12686a88280a053a99f59305ed92ff0374dc \
-                        sha256  f65f1e060318cbdeef493aea18e42f2814c35ef4d33f2c95747b00f00d544d6f \
-                        size    1133802
+checksums               rmd160  a735305895754d503a71c03a71225513c0d55ddc \
+                        sha256  5599d4351b716e1dc8716b3ddd5855fddbf7a48f47558e04648a8a731cb39637 \
+                        size    1134780
 
 # remove set deployment target and hard-coded RPATH setting
 patchfiles              patch-faudio-remove-deployment-target.diff
 
 configure.args          -DGSTREAMER=OFF \
                         -DBUILD_UTILS=OFF \
                         -DBUILD_TESTS=ON \
-                        -DXNASONG=ON
+                        -DXNASONG=ON \
+                        -DCMAKE_INSTALL_INCLUDEDIR=include/FAudio
 
```

Comment:
that is an unusual thing to add to a port. I have never seen that added to a Portfile before, so it feels like the wrong thing to do. What's up here?

Reply:
Usually, this would be done in the cmakelists.txt file for the software itself, I believe. Any software on any system that uses FAudio would expect to look in the same area for it's include files, it would see. If they are supposed to be tucked away in an FAudio subfolder, that's fine, but ... MacPorts' install of FAudio should be the same / similar to all the other systems install of it... (ergo the CMakeLists.txt comment)...

Reply:
Instead of just dumping the headers into `${prefix}/include`, I've moved them into `${prefix}/include/FAudio`

It seems a little odd but that's the command that accomplishes it for FAudio

Reply:
> Usually, this would be done in the cmakelists.txt file for the software itself, I believe. Any software on any system that uses FAudio would expect to look in the same area for it's include files, it would seem. If they are supposed to be tucked away in an FAudio subfolder, that's fine, but ... MacPorts' install of FAudio should be the same / similar to all the other systems install of it... (ergo the CMakeLists.txt comment)...

I've moved the headers to match what Linux distros are doing, the only thing that currently uses FAudio outside of FNA is wine and this calls pkgconfig for FAudio

Reply:
every linux distro can't be manually moving them there or specing them to be moved there on an individual basis with a cmake override tho. The software is supposed to set this itself (ie in CMakeLists.txt). It's odd to do it in a Portfile...

Reply:
> every linux distro can't be manually moving them there or specing them to be moved there on an individual basis with a cmake override tho. The software is supposed to set this itself (ie in CMakeLists.txt). It's odd to do it in a Portfile...

Previously each distro was also adding there own pc file and something that wine expected, yet FAudio only now added pc file creation in this patch

Reply:
> Instead of just dumping the headers into `${prefix}/include`, I've moved them into `${prefix}/include/FAudio`

Why? What problem does this solve? What's the upstream bug report URL about that problem?

Reply:
> > Instead of just dumping the headers into `${prefix}/include`, I've moved them into `${prefix}/include/FAudio`
> 
> Why? What problem does this solve? What's the upstream bug report URL about that problem?

I brought this up during the pkgconfig issue but it was left up to each distro to handle the location, as I see Linux distros using this location I configured it to match those. It doesn't resolve nor create any issues currently as only Wine make use of FAudio currently.